common/proto-conv/src/util.rs

```diff
@@ -14,7 +14,7 @@
 
 use crate::Incompatible;
 
-pub const VER: u64 = 1;
+pub const VER: u64 = 2;
```

Comment:
We are going to increase the VER of proto in this PR?

Reply:
what does increase the VER here means? 👀

Reply:
It means it is another new version of meta-types, since it introduces `share` related types and has been different from the previous version of meta-types.

A new version databend-query that depends on `share` data can set `MIN_COMPATIBLE=2` so that it will read `share` data happily and will refuse to read metadata with `VER=1` that does not have `share` related data.

Reply:
> We are going to increase the VER of proto in this PR?

Hmm... it would be nice if every time changing meta types there is a new `VER` value.
This way we'll be able to get rid of old version meta types in fine-grained progress.